docs/src/literate/ns_vs_diffeq.jl

```diff
@@ -0,0 +1,507 @@
+            end
+        end
+
+        # The last step in the element loop is to assemble `Kₑ` and `fe`
+        # into the global `K` and `f` with `assemble!`.
+        #+
+        assemble!(stiffness_assembler, celldofs(cell), Kₑ)
+    end
+    return K
+end;
+
+# ### Solution of the semi-discretized system via DifferentialEquations.jl
+# First we assemble the linear portions for efficiency. These matrices are
+# assumed to be constant over time.
+T = 10.0
+Δt₀ = 0.01
+Δt_save = 0.1
+
+M = create_sparsity_pattern(dh);
+M = assemble_mass_matrix(cellvalues_v, cellvalues_p, M, dh);
+
+K = create_sparsity_pattern(dh);
+K = assemble_stokes_matrix(cellvalues_v, cellvalues_p, ν, K, dh);
+
+# These are our initial conditions. We start from the zero solution, because it
+# is trivially admissible if the Dirichlet conditions are zero everywhere on the
+# Dirichlet boundary for t=0. Note that the time stepper is also doing fine if the
+# Dirichlet condition is non-zero and not too pathological.
+u₀ = zeros(ndofs(dh))
+apply!(u₀, ch);
+
+# DifferentialEquations assumes dense matrices by default, which is not
+# feasible for semi-discretization of finite element models. We communicate
+# that a sparse matrix with specified pattern should be utilized through the
+# `jac_prototyp` argument. It is simple to see that the Jacobian and the
+# stiffness matrix share the same sparsity pattern, since they share the
+# same relation between trial and test functions.
+jac_sparsity = sparse(K);
+
+# To apply the nonlinear portion of the Navier-Stokes problem we simply hand
+# over the dof handler and cell values to the right hand side as a parameter.
+# Further the pre-assembled linear part (which is time independent) is
+# passed to save some runtime. To apply the time-dependent Dirichlet BCs, we
+# also hand over the constraint handler.
+# The basic idea to apply the Dirichlet BCs consistently is that we copy the
+# current solution `u`, apply the Dirichlet BCs on the copy, evaluate the
+# discretized right hand side of the Navier-Stokes equations with this vector
+# and finally set the rhs to zero on every constraint. This way we obtain a
+# correct solution for all dofs which are not Dirichlet constrained. These
+# dofs are then corrected in a post-processing step, when evaluating the
+# solution vector at specific time points.
+# It should be finally noted that this *trick does not work* out of the box
+# *for constraining algebraic portion* of the DAE, i.e. if we would like to
+# put a Dirichlet BC on pressure dofs. Here we have to set "$d_t p_i = p_i$" instead
+# of "$d_t p_i = 0$", because otherwise the equation system gets singular. This
+# is obvious when we also take into account that our mass matrix is zero for these
+# dofs, such that we obtain the equation $0 \cdot d_t p_i = p_i$.
+struct RHSparams
+    K::SparseMatrixCSC
+    ch::ConstraintHandler
+    dh::DofHandler
+    cellvalues_v::CellVectorValues
+end
+p = RHSparams(K, ch, dh, cellvalues_v)
```

Comment:
```suggestion
p = RHSparams(K, ch, dh, cellvalues_v)

```
white line, otherwise function declaration and RHSparams stuff is in one Documenter code block

Reply:
![grafik](https://user-images.githubusercontent.com/27497290/135423859-f797a351-807e-4e4c-bd87-24bb527ec8f0.png)